examples/spectrum/src/wasm/frontend.rs

```diff
@@ -84,10 +86,10 @@ pub fn Spectrum(fg_handle: FlowgraphHandle) -> impl IntoView {
 
     view! {
         <div class="text-white">
-            <button class="bg-slate-600 hover:bg-slate-800 rounded p-2 m-4" on:click=ctrl_click>Show/Hide Controlls</button>
+            <button class="p-2 m-4 rounded bg-slate-600 hover:bg-slate-800" on:click=ctrl_click>Show/Hide Controlls</button>
```

Comment:
I assume you have a plugin that does that reordering for you. Do you have a recommendation?

Reply:
lepotsfmt:`cargo install leptosfmt`  
rust-analyzer.toml:`[rustfmt]
overrideCommand = ["leptosfmt", "--stdin", "--rustfmt"]`

Reply:
Thanks, didn't know that this will also take care of Tailwind classes.